apps/meteor/tests/end-to-end/api/livechat/12-priorites.ts

```diff
@@ -21,23 +21,35 @@ import { IS_EE } from '../../../e2e/config/constants';
 			.then(done);
 	});
 
+	const removePermissions = async (perms: string[]) => {
+		await updateManyPermissions(Object.fromEntries(perms.map((name) => [name, []])));
+	};
+	const addPermissions = async (perms: { [key: string]: string[] }) => {
+		await updateManyPermissions(perms);
+	};
```

Comment:
You can put this one inside permissions.helper too 👀 , since are functions we can use in other places

Reply:
done